packages/elements/src/button/__test__/button.test.js

```diff
@@ -223,22 +219,22 @@ describe('button/Button', () => {
 
 
     describe('should pass a11y testing in toggle mode', () => {
-      it('when button is not press', async () => {      
+      it('when button is not press', async () => {
```

Comment:
```suggestion
      it('when button is not pressed', async () => {
```

Reply:
This change is not part of this MR

Reply:
You did this to me in the perf change! 😆

Reply:
Oh no ​🗿​😟​🗿​

Reply:
Fixed